### PR TITLE
docs: minor updates to README.md and package.json

### PR DESCRIPTION
**Pull request by piperchester**
### What

This change includes:
- Fixing casing of "Javascript" to "JavaScript"
- Adding code highlighting
- Alphabetizing the keywords in `package.json`

### Why

I'm really excited about this project. Was just browsing some of the source and noticed the low hanging fruit.

### SDK Release Checklist

- [N/A] Have you added an integration test for the changes?
- [N/A] Have you built the gem locally and made queries against it successfully?
- [X] Did you update the changelog?
- [X] Did you bump the package version?
- [N/A] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.2.5] - 2020-11-28
```
Comment (biglovisa) [inline CHANGELOG.md:10]: Since there are no changes to functionality, we won't need to submit a release with this change.

**Review by biglovisa — CHANGES_REQUESTED** [review]
Thanks for the PR, @piperchester! 🙌🏻

File: CHANGELOG.md (hunk 1 above)
Comment (piperchester) [reply CHANGELOG.md]: Sounds good! I backed out all version bump changes (including the bump in `package.json`). Do we still want to bump `package.json` to v1.2.5 or is that sufficient?

Comment (biglovisa) [reply CHANGELOG.md]: Thanks! Keeping it at v1.2.4 should be good 👍🏻

**Review by biglovisa — APPROVED** [review]

**Pull request merged**